### DISCUSSION TITLE
Mulighet til å koordinere logoer

### DISCUSSION THREAD

**Discussion by PeterRydberg** [entur/tavla#315] [Ideas]
Akkurat nå laster man opp tavlelogoer individuelt til hver tavle. Dette fungerer greit når man bare skal ha en-to tavler knyttet til seg, men det kan bli vanskelig om man er bedrift og har veldig mange tavler på gang. Hvis en logo noen gang oppdateres, må administratoren gå inn på hver enkelt tavle og laste opp en ny versjon. Derfor burde det eksistere et system som gjør at ulike tavler kan kobles til samme logo i backend. Dette vil jo også være besparende på lagring i Firebase!

Nå vet jeg ikke nøyaktig hvordan dette bør se ut, men jeg tenker det er naturlig at logosiden på adminpanelet har et felt med "Dine logoer" (som er knyttet til din bruker) hvor man kan laste opp/fjerne/endre hver enkelt logo. Herfra kan brukeren velge hvilken av disse logoene som skal brukes for tavla som er i ferd med å endres. Endringer på én logo vil da endres på alle tavlene som bruker den.

Det her blir jo en litt større feature, men det kan være lurt på sikt! Spesielt når Tavla blir brukt på bussholdeplasser og slikt.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
